sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1898,10 +1926,19 @@ class SGDRegressor(BaseSGDRegressor):
           Each time n_iter_no_change consecutive epochs fail to decrease the
           training loss by tol or fail to increase validation score by tol if
           early_stopping is True, the current learning rate is divided by 5.
+        - 'pa1': passive-aggressive algorithm 1, see [1]_. Only with
+          `loss='epsilon_insensitive'`.
+          Update is `w += eta y x` with `eta = min(PA_C, loss/||x||**2)`.
+        - 'pa2': passive-aggressive algorithm 2, see [1]_. Only with
+          `loss='squared_epsilon_insensitive'`.
```

Comment:
It looks like we raise an error in the base class if the loss is not `epsilon_insensitive`. Do we need to adjust the validation for this case too?

Reply:
Same as for the classification loss hinge, see above.